packages/server/src/lib/access-helpers.js

```diff
@@ -1,19 +1,19 @@
-const { getUser } = require('../db');
-
-function isPartOfAgency(agencies, agencyId) {
-    return agencies.find((s) => s.id === Number(agencyId));
-}
+const { getUser, inTenant } = require('../db');
 
 /**
  * Determine if a user is authorized for an agency.
  *
- * @param {Number} userId
+ * @param {Object} user
  * @param {Number} agencyId
- * @returns {Boolean} true if the agency is the user's or a descendant; false otherwise
- */
+ * @returns {Boolean} true if the agency is in the same tenant as the user
+ * */
+async function isUserAuthorized(user, agencyId) {
```

Comment:
Naming of the parameter here and the way it's passed through to `inTenant` doesn't make it clear that it can be a single ID or a list -- as noted elsewhere, would prefer to just make it always a list and name it `agencyIds`.

Also: I'd consider renaming this function to be a bit more explicit (e.g. `isUserAuthorizedForAgency`) since it's now used in many more places outside this file where the context is less obvious about what authorization means.